xml/System.Diagnostics/DiagnosticListener.xml

```diff
@@ -107,7 +107,7 @@
       </ReturnValue>
       <Parameters />
       <Docs>
-        <summary>To be added.</summary>
+        <summary>Clean up the NotificationListeners. Notification listeners do NOT DIE ON THEIR OWN because they are in a global list (for discoverability).  You must dispose them explicitly. Note that we do not do the Dispose(bool) pattern because we frankly don't want to support subclasses that have non-managed state.</summary>
```

Comment:
The all-caps here seems a little aggressive

Reply:
The word "frankly" also seems a little passive-aggressive

Reply:
Should be `dispose of them` rather than `dispose them`

Reply:
```suggestion
        <summary>Clean up the NotificationListeners. Notification listeners do not die on their own because they are in a global list (for discoverability). You must dispose of them explicitly. Note that we do not do the Dispose(bool) pattern because we don't want to support subclasses that have non-managed state.</summary>
```